df_py/util/constants.py

```diff
@@ -39,3 +39,4 @@
 # predictoor
 MIN_PREDICTIONS = 1818
 DEPLOYER_ADDRS: List[str] = []
+PREDICTOOR_RELEASE_WEEK = 42  # change me when released
```

Comment:
I (still) do not agree with adding variables in constants.py, but if it's the only way, I guess it's fine.